### DISCUSSION TITLE
How to preview component with different token sets at once

### DISCUSSION THREAD

**Discussion by NamekMaster** [six7/figma-tokens#1473] [Q&A]
For example, I have two token sets, light and dark. When I select light, my component switch to light, and when I select dark, the component switch to dark. I'd like to show my component both light and dark at once for developers without permission to edit the file or use plugins. How do I achieve this ?

**Comment by six7** [comment]
You can choose `Apply to selection` and then select the first frame, set that to Light. Then select the next frame, and set that to Dark.

We'll work on a way to lock frames to a certain theme.
